### PR TITLE
add phone number field and fix minor ui in privacy policy page

### PR DESCRIPTION
**Pull request by mingbeam**
### Problems
 

early access
- add phone number field
- use api to subscribe
- change ui to match the added field (also for responsive version)
- MailchimpApiForm is mainly from earlybirds SignupForm
- its css is mainly from MailchimpForm.module.css
- make animation in phone version comes faster using fadeInFast instead of fadeIn (has less delay)

privacy policy
- wider sidebar and have right padding
- less left padding on the content side
- rename file to privacy.tsx
 
mailchimpSignupForm
- use margin instead of gap
- address comments about css
- add type tel in phone input field
- remove unused code

### How would the reviewers verify the solutions
Screenshot (1383)

Screenshot (1382)

Screenshot (1384)

### Preflight checklist

- [ ] **Manual testing** has been performed according to the acceptance criteria.
- [ ] _(heavy style change)_ The solution has been tested on the major browser (Chrome, Firefox, Safari, Microsoft Edge).

### CODE REVIEW

File: components/MailchimpApiForm.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                )}
+                disabled={formik.isSubmitting}
+              />
+              {formik.errors.email && formik.touched.email && (
+                <ErrorStateMessage name="email" />
+              )}
+            </InputFieldWrapper>
+
+            <InputFieldWrapper>
+              <Field
```
Comment (oukunanan) [inline components/MailchimpApiForm.tsx:72]: What is the `input type` of the phone number 🤔? If open in the mobile phone. the keyboard should show only the number.

File: components/MailchimpApiForm.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+              {formik.errors.phoneNumber && formik.touched.phoneNumber && (
+                <ErrorStateMessage name="phoneNumber" />
+              )}
+            </InputFieldWrapper>
+
+            <button
+              type="submit"
+              className={classNames(
+                styles.submit_button,
+                formik.isSubmitting && styles.submit_button_submitting
```
Comment (oukunanan) [inline components/MailchimpApiForm.tsx:93]: Use `:disabled` pseudo-selector

File: styles/Home.module.css [hunk 3]
```diff
@@ -173,6 +174,10 @@
     margin-right: 16px;
   }
 
+  .contentWrapper {
+    overflow: scroll;
```
Comment (oukunanan) [inline styles/Home.module.css:178]: I don't like use `overflow: scroll` here. it should affect only the privacy page but keep it for now. Please be aware of it.

Comment (mingbeam) [reply styles/Home.module.css:178]: noted krub.

File: components/MailchimpSignupForm.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    .email('Please enter a valid email address')
+    .required('Required'),
+  phoneNumber: Yup.string()
+    .matches(phoneRegExp, 'Please enter a valid phone number')
+    .min(8, 'Must be between 8-10 characters')
+    .max(10, 'Must be between 8-10 characters')
+    .required('Required'),
+})
+
+export default function MailchimpApiForm() {
```
Comment (oukunanan) [inline components/MailchimpSignupForm.tsx:26]: Sync up with the file name

File: components/MailchimpSignupForm.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+      <SuccessModal
+        isOpen={successModalVisible}
+        closeModal={() => setSuccessModalVisible(false)}
+      />
+    </>
+  )
+}
+
+function InputFieldWrapper(props: {
+  className?: string
```
Comment (oukunanan) [inline components/MailchimpSignupForm.tsx:114]: Is there the other place using `className`? if not please remove it.

File: styles/globals.css [hunk 6]
```diff
@@ -66,6 +66,13 @@ button {
   font-family: 'Lexend Deca';
 }
 
+button {
+  border: 2px solid #3a52b8;
+  border-radius: 6px;
+  background-color: #3a52b8;
+  cursor: pointer;
+}
```
Comment (oukunanan) [inline styles/globals.css:74]: Please double-check in the privacy page if using any button

**Review by oukunanan — APPROVED** [review]
Approved with comments

File: components/MailchimpSignupForm.tsx (hunk 5 above)
Comment (mingbeam) [reply components/MailchimpSignupForm.tsx:114]: removed

File: styles/globals.css (hunk 6 above)
Comment (mingbeam) [reply styles/globals.css:74]: checked, no button in privacy page

**Pull request merged**